pkg/util/dynamichelper/gvrresolver_test.go

```diff
@@ -195,17 +199,35 @@ func TestFindGVR(t *testing.T) {
 		},
 	}
 
-	for _, tt := range tests {
+	for index, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			r := &gvrResolver{apiresources: tt.resources}
 
 			got, err := r.Resolve(tt.kind, "")
-			if !reflect.DeepEqual(err, tt.wantErr) {
-				t.Errorf("got: %#v, expected: %#v", err, tt.wantErr)
+			if err != nil {
+				for _, i := range deep.Equal(err.Error(), tt.wantErr.Error()) {
+					t.Error(i)
+				}
+				return
 			}
+
 			if !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("got: %#v, expected: %#v", got, tt.want)
+				t.Errorf("%d Result: got: %#v, expected: %#v", index, got, tt.want)
```

Comment:
Is adding the index helpful? If we fail here, isn't it already telling us "you're on test X, got Y expected Z"?

Reply:
During testing if a nil pointer exception occurs it doesn’t show which test actually fails which is why I added the indexing.